Lib/trufont/controls/glyphDialogs.py

```diff
@@ -162,17 +163,26 @@ def getLayerAndAction(cls, parent, currentGlyph):
         return (newLayer, action, result)
 
 
-class RenameDialog(QDialog):
+class EditDialog(QDialog):
 
-    def __init__(self, parent=None):
+    def __init__(self, parent=None, item=None):
         super().__init__(parent)
         self.setWindowModality(Qt.WindowModal)
-        self.setWindowTitle(self.tr("Rename…"))
+        self.setWindowTitle(self.tr("Edit…"))
 
         nameLabel = QLabel(self.tr("Name:"), self)
         self.nameEdit = QLineEdit(self)
         self.nameEdit.setFocus(Qt.OtherFocusReason)
 
+        validator = QDoubleValidator(self)
+        validator.setLocale(QLocale.c())
```

Comment:
Is this to input numbers in the user locale's format? If that's needed it should probably be done consistently throughout because I don't think that's done anywhere else at the moment.

Reply:
I just wanted to restrict the input to numbers, if you know a better way I can use it.

Reply:
QDoubleValidator forces the input to be a number yes, but the locale thing isn't called anywhere else we use QDoubleValidator. The current locale should be set by default already. Why do you call setLocale?

Reply:
We don’t currently use `QDoubleValidator` anywhere else, and I wasn’t sure the rest of the code can handle locale-specific float formatting (but if you use French locale yourself then the code probably handles it 😄)

Reply:
Oh, I remember now. I use an Arabic locale, and without forcing C locale here the input box won’t accept entering `.` as decimal point and would only accept the Arabic decimal separator which actually is not accessible from Arabic keyboard layout.